app/assets/javascripts/application.js

```diff
@@ -12,5 +12,6 @@
 //
 //= require jquery
 //= require jquery_ujs
-//= require turbolinks
-//= require_tree .
+//= require d3
+//= require topojson
+//= require_tree .
```

Comment:
Set up your text editor to always add new lines at the EOF. Just a good practice to have them.